lib/cylc/cfgspec/globalcfg.py

```diff
@@ -509,20 +511,21 @@ def get_host_item(self, item, host=None, owner=None, replace=False):
                         host_key = cfg_host
                         break
         modify_dirs = False
-        if host_key:
+        if host_key is not None:
             # entry exists, any unset items under it have already
             # defaulted to modified localhost values (see site cfgspec)
             value = cfg['hosts'][host_key][item]
         else:
             # no entry so default to localhost and modify appropriately
             value = cfg['hosts']['localhost'][item]
             modify_dirs = True
-
-        if value and ('directory' in item) and (
-                modify_dirs or owner != USER or replace):
-            # replace local home dir with $HOME for evaluation on other host
-            value = value.replace(os.environ['HOME'], '$HOME')
-
+        if value is not None and 'directory' in item:
+            if replace_home and (modify_dirs or owner != USER):
+                # replace local home dir with $HOME for eval'n on other host
+                value = value.replace(os.environ['HOME'], '$HOME')
+            elif owner != USER:
+                # replace USER with owner for direct access via local filesys
+                value = value.replace('/%s/' % USER, '/%s/' % owner)
```

Comment:
(in which case we could assume `~user/cylc-run/` rather than replacing `/me/` with `/you/` in the path - that would work so long as `~you` has not configured a non-standard cylc-run location.)